app/uk/gov/hmrc/gform/formtemplate/FormTemplatesControllerRequestHandler.scala

```diff
@@ -157,7 +162,35 @@ object FormTemplatesControllerRequestHandler {
         case (None, Some(_), None, None)       => JsSuccess(())
       }
 
-    sectionValidations andKeep jsonValue.transform(
+    val dmsFormIdValidations: JsResult[Unit] =
+      (jsonValue \ "destinations").toOption match {
+        case Some(id) =>
```

Comment:
the variable id could be renamed to destinationsJsArray or something else

Reply:
I believe variable id **dmsFormIdValidations** is more apporiprate. It's inline with othera   **sectionValidations andKeep dmsFormIdValidations andKeep jsonValue.transform(**

Reply:
case Some(id), here id represents an array of destinations and not just a single id. it cause a bit of confusion when I read it for a the first time.